composer.json

```
@@ -83,6 +83,7 @@
         ],
         "cs:check": "@php php-cs-fixer fix --dry-run --diff",
         "cs:fix": "@php php-cs-fixer fix",
+        "cs:fix:para": "echo '🔍 Will run in batches of 50 files.'; php php-cs-fixer list-files --config=.php-cs-fixer.dist.php | xargs -n 50 -P 8 php php-cs-fixer fix --config=.php-cs-fixer.dist.php --path-mode intersection 2> /dev/null",
```

```suggestion
        "cs:fix:parallel": "if [[ -f .php-cs-fixer.php ]]; then FIXER_CONFIG=.php-cs-fixer.php; else FIXER_CONFIG=.php-cs-fixer.dist.php; fi; php php-cs-fixer list-files | xargs -n 50 -P 8 php php-cs-fixer fix --config=$FIXER_CONFIG --path-mode intersection 2> /dev/null",
```

- Support for fine-tuned local config
- `--config` is not needed for `list-files` (autodetected)
- "🔍 Will run in batches of 50 files." should be a part of `scripts-descriptions` which is missing now
- full `cs:fix:parallel` is more verbose and descriptive, but it's possible to use `cs:fix:para` anyway because Composer resolves closest matching names (it's even possible to use `c:f:p` to run it)

---

FYI: it must be in one command, because using `[]` and setting `FIXER_CONFIG` in separate command does not work, because it's not available within `xargs` on later stage.

---

thanks for suggestion. let me incorporate it, yet for now I would like to keep the explicit config for `list-files`, for consistency with explicit config for `fix`. On long run, i hope we can get rid of the need to pass it for `fix` command.

btw, thanks for `c:f:p` hint